### DISCUSSION TITLE
How do I get this working on Ubuntu?

### DISCUSSION THREAD

**Discussion by fedorvaneldijk** [bladecoder/bladecoder-adventure-engine#61] [General]
I shared my problem as an issue, but maybe this is a better place: did anyone get this running on Linux?

**Comment by fedorvaneldijk** [comment]
Is this system still alive?

**Reply by bladecoder** [reply]
Yes, still alive :-)

Your problem is due to your JDK version. You need to use JDK 8 and probably you are using a newer version, downgrade to JDK 8 and be sure to use JDK and not JRE.

The next release will work with recents JDKs.

**Reply by virtualars** [reply]
I also use the latest version of jdk and since i make apps in kotlin and flutter i can't downgrade, what is the timing for the update?
